lib/devices/ios/ios.js

```diff
@@ -372,7 +354,7 @@ IOS.prototype.makeInstruments = function (cb) {
   var bootstrap = prepareBootstrap({
     sock: this.sock,
     interKeyDelay: this.args.interKeyDelay,
-    justLoopInfinitely: this.shouldIgnoreInstrumentsExit(),
+    justLoopInfinitely: false,
```

Comment:
@jlipps I decided to leave this here, since bootstrap.js doesn't need to change its interface. Do you think we should remove it completely?